SECURITY.md

```diff
@@ -0,0 +1,36 @@
+# Security policy
+
+## Supported versions
+
+All ops 2.x versions released within the last year are currently supported with security updates.
+
+## Reporting a vulnerability
+
+Please provide a description of the issue, the steps you took to
+create the issue, affected versions, and, if known, mitigations for
+the issue.
+
+The easiest way to report a security issue is through
+[GitHub](https://github.com/canonical/operator/security/advisories/new). See
+[Privately reporting a security
+vulnerability](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability)
+for instructions.
```

Comment:
What about this?

```suggestion
for instructions on reporting using GitHub's security advisory feature.
```

Reply:
I guess that also works.

I suppose a URL would also work instead of a link -- it's pretty clear what it is and where it leads.